addons/binding/org.openhab.binding.cbus/src/main/java/org/openhab/binding/cbus/internal/cgate/CGateConnectException.java

```diff
@@ -0,0 +1,30 @@
+/**
+ *  CGateInterface - A library to allow interaction with Clipsal C-Gate.
```

Comment:
Wow, you are quick @daveoxley, many thanks! So over to @neohusky now (unless @talltechdude is still/again around)?